app/db.server.ts

```diff
@@ -1,15 +1,15 @@
-import knex from "knex";
-import knexfile from "~/../knexfile";
+import SQLite from "better-sqlite3";
+import { Kysely, SqliteDialect } from "kysely";
+import type { DB } from "kysely-codegen";
+
 export { SqliteError } from "better-sqlite3";
 
-const environment = process.env.NODE_ENV || ("development" as const);
-// @ts-nocheck
-const config: {
-  client: string;
-  connection: {
-    filename: string;
-  };
-  useNullAsDefault: boolean;
-} = (knexfile as any)[environment];
-
-export default knex(config);
+export const dialect = new SqliteDialect({
+  database: new SQLite(process.env.DATABASE_URL),
```

Comment:
This will probably need to get added to [helpers/env.ts](https://github.com/DanielFGray/rf-mod-bot/blob/kysely/app/helpers/env.ts)